### ISSUE TITLE
[indent] Multiline property values treated as properties in interfaces

### ISSUE THREAD

**Issue by stevenhair** [typescript-eslint/typescript-eslint#3381]
- [X] I have tried restarting my IDE and the issue persists.
- [X] I have updated to the latest version of the packages.
- [X] I have [read the FAQ](https://github.com/typescript-eslint/typescript-eslint/blob/master/docs/getting-started/linting/FAQ.md) and my problem is not listed.

**Repro**

```JSON
{
  "rules": {
    "@typescript-eslint/indent": "error"
  }
}
```

```TS
interface Foo {
    foo:
        'foo';
}

const foo = {
    foo:
        'foo',
};
```

**Expected Result**

Both the interface and the object indentation should be valid (or at least treated the same way)

**Actual Result**

ESLint does not error on the object indentation, but it provides this error for the interface: `Expected indentation of 4 spaces but found 8`

**Additional Info**

ESLint doesn't seem to care how far the Object property value is indented.

**Versions**

| package                            | version |
| ---------------------------------- | ------- |
| `@typescript-eslint/eslint-plugin` | `4.23.0` |
| `@typescript-eslint/parser`        | `4.23.0` |
| `TypeScript`                       | `4.2.4` |
| `ESLint`                           | `7.26.0` |
| `node`                             | `14.16.1` |

**Comment by bradzacher** [comment]
please use the search or look at the pinned issues next time.
https://github.com/typescript-eslint/typescript-eslint/issues/1824

**Comment by stevenhair** [comment]
Apologies - I was searching for my specific issue rather than a general issue with the indent rule.
